src/csharp/Microsoft.Spark.E2ETest/UdfTests/UdfComplexTypesTests.cs

```diff
@@ -244,7 +244,7 @@ public void TestUdfWithReturnAsRowType()
                 }
             }
 
-            // Nested GenericRow
+            // Nested GenericRows.

```

Comment:
// Test UDF that returns a Row object.

Reply:
I have changed to `// Test UDF that returns a nested Row object.`.